cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +32,27 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('should not provide an ability to log in with wrong email', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    signInPage.typeEmail(wrongUser.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+
+    signInPage.verifyEmailError();
+
+  });
+
+  it('should not provide an ability to log in with wrong password', () => {
+
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
```

Comment:
The `cy.register` command is used here to register the user. Ensure that this registration is necessary for each test, as it might be redundant if the user is already registered in the `beforeEach` hook.